contracts/MarketMaker.sol

```diff
@@ -174,12 +188,12 @@ contract MarketMaker is Ownable, IERC1155TokenReceiver {
                 collateralToken.approve(address(pmSystem), uint(outcomeTokenNetCost))
             );
 
-            pmSystem.splitPosition(collateralToken, bytes32(0), conditionId, partition, uint(outcomeTokenNetCost));
+            splitPositionThroughAllConditions(uint(outcomeTokenNetCost), conditionIds.length, 0);
```

Comment:
I think splitting should also be affected by the `payoutAtomicTokens` parameter in this case. Also, the frontend is not coded to take advantage of this parameter yet.

Reply:
I've thought about that. I cant quite picture how you envision that working? As, splitting already gets paid in `AtomicOutcomeTokens`.